File: WineriesApp/src/WineriesApp.Services/Models/AddReview.cs
using WineriesApp.DataContext.Enums;

namespace WineriesApp.Services.Models;

public class AddReview
{
    public double Rating { get; set; }
    
    public string? Comment { get; set; }
    
    public ReviewEntityType EntityType { get; set; }
    
    public Guid EntityId { get; set; }
}

File: WineriesApp/src/WineriesApp.UI/src/models/AddReview.ts
import { ReviewEntityType } from "../enums/ReviewEntityType";

export class AddReview {
    public Rating: number;
    public Comment: string;
    public EntityType: ReviewEntityType;
    public EntityId: string;

    public constructor(rating: number, comment: string, entityType: ReviewEntityType, entityId: string) {
        this.Rating = rating;
        this.Comment = comment;
        this.EntityType = entityType;
        this.EntityId = entityId;
    }
}